src/platform/monitoring/DowntimeNotification/README.md

```diff
@@ -0,0 +1,19 @@
+# Downtime Notifications
+Downtime Notications provide a method of handling service outages in front-end applications. For example, if your application _My App_ leveraged a service called _Fancy Service_, and _Fancy Service_ was known to be offline for maintenance every now and then, it would be desirable to show a message to the user explaining that _My App_ is unavailable at this time so that users aren't given the impression that they are running into unexpected errors and so that _Fancy Service_ isn't needlessly hit. This should also be easy for engineers to do, without requiring any deployments to publish/unpublish the messaging.
+
+This was implemented in Downtime Notifications via a React component intended to wrap an application's entry point. After a user lands on the route for that React app, the Downtime Notification React component will fetch the maintenance windows from the VA API, which are registered in [Pager Duty](https://dsva.pagerduty.com/maintenance_windows). If there are any active maintenance windows for the application's registered dependencies, then instead of rendering the application as usual, the application will render a message explaining to the user that the application is undergoing maintenance and the timeframe for which it will last.
+
+## Implementation
```

Comment:
Implementation suggests that this is a description of the inner workings of the downtime notification. 

How to use or how to set up or how to implement might be clearer.